src/SecurityAlertIssue.php

```diff
@@ -91,10 +98,13 @@ public function __construct(array $data)
      */
     public function uniqueId(): string
     {
+        // If there is no safe version we use the GHSA ID as identifier.
+        $identifier = $this->safeVersion ?? $this->id;
```

Comment:
This means that if a fix becomes available, it'll create a new ticket. Which is good, but shouldn't this side effect be mentioned in a comment?